crates/l2/contracts/src/l1/OnChainProposer.sol

```diff
@@ -176,6 +159,23 @@ contract OnChainProposer is
         OwnableUpgradeable.__Ownable_init(owner);
     }
 
+    /// @inheritdoc IOnChainProposer
+    function initializeBridgeAddress(address bridge) public onlyOwner {
```

Comment:
Wouldn't we need to do the same for the verifiers?

Reply:
Why? The bridge case is special since both needs the address of the other so it's a circular dependency, but in the case of the verifiers, only the sequencer needs to know their addresses. It would be enough to deploy the verifiers before (in case they are not already deployed)

Reply:
Nvm, now I understand the reason correctly.